EventFilter/L1TRawToDigi/python/gtStage2Digis_cfi.py

```diff
@@ -5,7 +5,7 @@
     Setup           = cms.string("stage2::GTSetup"),
     InputLabel      = cms.InputTag("rawDataCollector"),
     FedIds          = cms.vint32( 1404 ),
-    FWId            = cms.untracked.int32(2),
+    FWId            = cms.uint32(2),
     lenSlinkHeader  = cms.untracked.int32(8),
     lenSlinkTrailer = cms.untracked.int32(8),
```

Comment:
Indeed... but my attempt to changing some of the other ones broke HLT scripts.  So was postponing this until HLT/L1T integration step.